src/install/dependencies.rs

```diff
@@ -1,2 +1,2 @@
 pub const WASM_PACK_VERSION: &str = "0.9.1";
-pub const GENERATE_VERSION: &str = "0.5.0";
+pub const GENERATE_VERSION: &str = "0.5.1";
```

Comment:
I don't know the rules for incrementing these versions, @koeninger @xortive @nataliescottdavidson ?

Seems like adding a backwards-compatible feature would increment the minor version (`0.5.0` -> `0.6.0`) but I'm not sure.

Reply:
I think these refer to actual cargo-generate versions. The latest one is currently `0.5.2`

Reply:
Yes this brings in cargo generate 0.5.1 which allow to strip the `https://github.com/` part of the template url:
```
# 0.5.0
wrangler generate my-ts-project https://github.com/p-j/worker-eapi-template --branch=main
# 0.5.1
wrangler generate my-ts-project p-j/worker-eapi-template --branch=main
```

Since the PR was made cargo-generate 0.5.2 and 0.5.3 have been published.
Let me know if you want me to update the PR with any of those versions.

Reply:
looks like cargo-generate's CI isn't generating assets past 0.5.3, so going to revert the change I made